xcell/mappers/mapper_P18SMICA.py

```diff
@@ -27,7 +36,12 @@ def _get_hm_maps(self):
 
     def get_mask(self):
         if self.mask is None:
-            self.mask = np.ones(12*self.nside**2)
+            if self.file_mask is not None:
+                self.mask = hp.read_map(self.file_mask)
+                self.mask = hp.ud_grade(self.mask,
+                                        nside_out=self.nside)
+            else:
+                self.mask = np.ones(12*self.nside**2)
```

Comment:
Should we not be consistent in what we do with point source masks also for smica? It seems like here the only thing you allow for is the first map in the list of point source masks.

Reply:
that's true we can make it more general